src/components/atoms/SfRadio/README.md

```diff
@@ -0,0 +1,39 @@
+# SfRadio
+
+Component for simple group of radio buttons, pass an array get selected value via v-model
+
+## Props
```

Comment:
shold be docummented in component as we discussed, don't duplicate ;)

Reply:
https://github.com/DivanteLtd/storefront-ui/pull/91/commits/0d33e8d6f260893ffb7cb1d968477c6ef880989d